src/api/json/catalog.json

```diff
@@ -3083,6 +3083,19 @@
       ],
       "url": "https://json.schemastore.org/typingsrc.json"
     },
+    {
+      "name": "Ubuntu Server Autoinstall",
+      "description": "Settings file for Ubuntu Autoinstall",
+      "fileMatch": [
+        "user-data"
```

Comment:
There is no file extension. Is that on purpose?

Reply:
It is. There's no file extension for the file itself, as [per Ubuntu](https://ubuntu.com/server/docs/install/autoinstall-quickstart). Section is "Write your autoinstall config"